basicOptions.lua

```diff
@@ -332,28 +332,40 @@ fctfloatmodeDropdown:HookScript("OnEnter", function(self)
 end)
 fctfloatmodeDropdown:HookScript("OnLeave", GameTooltip_Hide)
 
+local revUVARINFO = {}
+for k, v in pairs(UVARINFO) do
+	revUVARINFO[v.cvar] = k
+end
+
+local function FCT_SetValue(self, checked)
+	addon:SetCVar(self.cvar, checked)
+	_G[revUVARINFO[self.cvar]] = checked and "1" or "0" -- update uvars
+	BlizzardOptionsPanel_UpdateCombatText()
```

Comment:
Have you tested if this is still true? http://git.tukui.org/Tukz/blizzard-user-interface/commit/da798f6f52d4721355b57bc0975d11dcea8ccfac#a4c43b2693ecb024350676b814a32e345e4fedb1_735_740